privacyidea/api/validate.py

```diff
@@ -111,6 +111,7 @@
 from privacyidea.lib.machine import list_machine_tokens
 from privacyidea.lib.applications.offline import MachineApplication
 import json
+import threading
```

Comment:
this isn't needed here anymore. Please also remove all the unused imports as well.